tfhe-zk-pok/src/proofs/mod.rs

```diff
@@ -141,5 +141,12 @@ pub const HASH_METADATA_LEN_BYTES: usize = 256;
 pub mod binary;
 pub mod index;
 pub mod pke;
+pub mod pke_v2 {
+    pub use super::pke_v2_impl::*;
+
+    pub use super::pke_v2_impl::crs_gen_cs as crs_gen;
+}
```

Comment:
what is the reason for the indirection here from pke_v2 to pke_v2_impl ?

Reply:
just cause i wanted a quick way to swap out pke for pke_v2 for benchmarks. i'll fix it

Reply:
got it

Reply:
i removed the indirection